charts/spinnaker/values.yaml

```
@@ -192,6 +201,12 @@ kubeConfig:
   - kube-public
   onlySpinnakerManaged:
     enabled: false
+  # A list of per-namespace account-related configuration. For the purpose of
+  # this chart, this only specifies the `namespace`. But this can be extended
+  # for own Helm templating to further configure Spinnaker (e.g. assigning
+  # permissions to the accounts).
+  namespacedAccounts:
```

I'm not sure `namespacedAccounts` is the most ideal name 🤔. OTOH, I do not have a good suggestion. The only thing that comes to mind right now is "enrichedAccounts" which is too generic and doesn't give a good context. 

Maybe `accountsWithNamespace(s)`? I don't really feel the verb `namespaced` 😅.

---

As discussed offline, this better conveys the message:

```suggestion
  namespacePartitionedAccounts: 
```

---

Done!